kie-drools-wb/kie-drools-wb-distribution-wars/src/main/assembly/component-kie-drools-wb-weblogic-12-common.xml

```diff
@@ -83,9 +70,12 @@
           <exclude>WEB-INF/lib/errai-jboss-as-support-*.jar</exclude>
           <!-- Errai 1.1+ CDI Compat -->
           <exclude>WEB-INF/lib/errai-weld-integration-*.jar</exclude>
+          <exclude>WEB-INF/lib/org.osgi.core-*.jar</exclude>
+          <exclude>WEB-INF/lib/javassist-*.jar</exclude>
+          <exclude>WEB-INF/lib/jgroups-*.jar</exclude>
```

Comment:
This is here to make sure the wars look the same after this change. TBH I am not quite sure it is correct. These jars are in EAP/WildFly wars, so either they are not needed there or they should be included in the WAS/WLS wars as well (as they are required by the apps to work correctly).